### PR TITLE
Wms tutorial

### PR DESCRIPTION
**Pull request by risehr**
Completed the Web Map Service Automated Tutorials.
It is tested on Windows and Linux both.
As of now, the cursor is visible in Screen Recordings in Windows but in Linux, the cursor is not visible in the tutorials recording (But if screenrecorder.py is run separately, it works in Linux too), I don't understand the reason.
Therefore, in Linux, I have made the cursor fixed in a particular position on the screen.
((The pyautogui.position() in screenrecorder.py in line 115 was throwing runtime error when the tutorial_wms.py is executed on Linux.))

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please use the template header on all created py files

File: tutorials/cursor.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import os
```
Comment (ReimarBauer) [inline tutorials/cursor.py:28]: please use our standard header on all created py files

**Comment by ReimarBauer** [comment]
Tried with only one screen, have you an idea what gets wrong?

```
Exception :'select to open control' button/option not found on the screen.

Exception : 'Server\Layers' button/option not found on the screen.

Exception : 'WMS URL' editbox button/option not found on the screen.

Exception : 'Get capabilities' button/option not found on the screen.

Exception : 'Divergence Layer' option not found on the screen.

Exception : 'Layer filter editbox' button/option not found on the screen.
Process Process-2:
Traceback (most recent call last):
  File "/home/reimar/miniconda3/envs/risehr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/reimar/miniconda3/envs/risehr/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/reimar/PycharmProjects/2021/risehr/MSS/tutorials/tutorial_wms.py", line 174, in automate_waypoints
    if x is not None and y is not None:
UnboundLocalError: local variable 'x' referenced before assignment

```

**Review by ReimarBauer — COMMENTED** [review]
> Tried with only one screen, have you an idea what gets wrong?
> 
> ```
> Exception :'select to open control' button/option not found on the screen.
> 
> Exception : 'Server\Layers' button/option not found on the screen.
> 
> Exception : 'WMS URL' editbox button/option not found on the screen.
> 
> Exception : 'Get capabilities' button/option not found on the screen.
> 
> Exception : 'Divergence Layer' option not found on the screen.
> 
> Exception : 'Layer filter editbox' button/option not found on the screen.
> Process Process-2:
> Traceback (most recent call last):
>   File "/home/reimar/miniconda3/envs/risehr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
>     self.run()
>   File "/home/reimar/miniconda3/envs/risehr/lib/python3.9/multiprocessing/process.py", line 108, in run
>     self._target(*self._args, **self._kwargs)
>   File "/home/reimar/PycharmProjects/2021/risehr/MSS/tutorials/tutorial_wms.py", line 174, in automate_waypoints
>     if x is not None and y is not None:
> UnboundLocalError: local variable 'x' referenced before assignment
> ```

ok, this happens because I had an individual mss_settings.json. When I remove it my linux system shows a typo on the url

types_sevens

**Comment by ReimarBauer** [comment]
Perhaps this is similiar that it relays on an US Keyboard

https://github.com/asweigart/pyautogui/issues/46

**Comment by ReimarBauer** [comment]
> Perhaps this is similiar that it relays on an US Keyboard
> 
> [asweigart/pyautogui#46](https://github.com/asweigart/pyautogui/issues/46)

By changing my german keyboar to an US Keyboard it works.

**Comment by risehr** [comment]
> please use the template header on all created py files

Okay. I will do that.
It's done.

**Comment by risehr** [comment]
> > Perhaps this is similiar that it relays on an US Keyboard
> > [asweigart/pyautogui#46](https://github.com/asweigart/pyautogui/issues/46)
> 
> By changing my german keyboar to an US Keyboard it works.

Ohh. Yes, I guess it works only for US keyboard layout.

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**